src/l500/l500-color.h

```diff
@@ -6,6 +6,7 @@
 #include <vector>
 #include <string>
 #include <map>
+#include <unordered_map>
```

Comment:
What's this for? I don't think I saw any new unordered_map...

Reply:
leftovers, removed.